### ISSUE TITLE
Battery not being detected by pijuice_cli, and not charging

### ISSUE THREAD

**Issue by Peytoncl** [PiSupply/PiJuice#854]
Im pretty sure i took the plastic covering off, unless theres more somewhere that i dont see? It flashed blue when i plug it in but then turns red and stays red, and says theres no battery connected.

**Comment by Peytoncl** [comment]
I am using the pijuice battery that came with my purchase

**Comment by tvoverbeek** [comment]
One thing to check in case your external power is connected to the Pi:
Is "GPIO input enabled checked? (On tha  Hat general tab in the GUi/ "General settings" tab in the CLI).
If not enabled the battery will not charge.

**Comment by Peytoncl** [comment]
Yes it says that the gpio input is enabled and charging is enabled

**Comment by Peytoncl** [comment]
i believe i have my battery connected, is theres something im missing? i just put my battery into the hat and dont do anythign else, should i ?

**Comment by johnathongotcher90** [comment]
my computer only works when plugged in and it wasnt like this when i got it
